src/core_modules/capture-core/metaDataStoreLoaders/index.js

```diff
@@ -1,3 +1,3 @@
 // @flow
-export { default as loadMetaDataAsync } from './baseLoader/metaDataLoader';
-export { default as loadSystemSettingsAsync } from './systemSettings/loadSystemSetttings';
+export { default as loadMetaData } from './baseLoader/metaDataLoader';
```

Comment:
I agere with most of your points here. Let's see:
1. > The first is about the default as loadMetaData. Can we say that we will have one name per function even if we do stuff like default as functionName? For example here I would rename the loadMetaDataAsync to loadMetaData inside the metaDataLoader.js file. This way the function has one name always. Is this sensible to you?

For the exact function mentioned here, loadMetaDataAsync is being renamed to loadMetData in a later PR in this PR-chain, so I agree with your point in principle. Sometimes the function needs renaming in the export due to the context though, but this is not the case here.

**While we are on the subject: I've stoppped exporting as default altogether. This makes refactoring easier since you have predictable names in the import. So I would suggest we keep this practice. Up for discussion if you disagree.**   

2. > The second has to do with the name of the folder metaDataLoader.js. Can we for simplicity name this folder loadMetaData.js as we do bellow with the loadSystemSetttings in the line bellow?

I guess you're talking about the file name here (not folder name)? If that is the case, I agree.

Reply:
> While we are on the subject: I've stoppped exporting as default altogether. This makes refactoring easier since you have predictable names in the import. So I would suggest we keep this practice. Up for discussion if you disagree.

Yes agree. 

> I guess you're talking about the file name here (not folder name)? If that is the case, I agree.

Yes I was talking about the file name :)